erpnext/payroll/doctype/salary_slip/salary_slip.py

```diff
@@ -63,7 +63,6 @@ def autoname(self):
 
 	def validate(self):
 		self.status = self.get_status()
-		validate_active_employee(self.employee)
```

Comment:
This validation should exist on Salary Slip too. Will fix the failing test. Any reason to completely remove it from Salary Slips?

Reply:
The validation is already done in `SalarySlip.validate_dates()`, so this was redundant.